cpp/models/abm/world.h

```diff
@@ -447,6 +470,75 @@ class World
     }
     /** @} */
 
+    /**
+     * @brief Flip activeness status of a person in the world.
+     * @param[in] person_id PersonId of Person whose activeness status is fipped.
+     */
+    void change_activeness(PersonId person_id)
+    {
+        m_activeness_statuses[person_id.get()] = !m_activeness_statuses[person_id.get()];
+    }
```

Comment:
This function requires already knowing the activeness state of a person. I prefer using the "set_person_active" function I propose above.